packages/ui/vitest.config.ts

```diff
@@ -6,6 +6,9 @@ import viteConfig from './vite.config'
 export default mergeConfig(
   viteConfig,
   defineConfig({
+    optimizeDeps: {
+      include: ['vue-router', 'splitpanes', 'd3-graph-controller', 'vue-virtual-scroller'],
+    },
```

Comment:
Not entirely sure about this, but without it Vite would trigger a reload and hang the test until the pipeline reaches the 30m timeout: https://github.com/vitest-dev/vitest/actions/runs/19538883275/job/55939684333?pr=8813